server/data/activeLduClient.js

```diff
@@ -19,4 +19,37 @@ module.exports = {
     const { rows } = await db.query(query)
     return parseInt(rows[0].count, 10) > 0
   },
+
+  async allActiveLdusInArea(probationAreaCode) {
+    const query = {
+      text: `SELECT ldu_code FROM active_local_delivery_units WHERE probation_area_code=$1`,
+      values: [probationAreaCode],
+    }
+
+    const { rows } = await db.query(query)
+    return rows
+  },
+
+  async updateWithActiveLdu(probationAreaCode, activeLduCodes) {
+    // flush then insert all the ldus in activeLduCodes array assigning the same probationAreaCode to each one
+
+    await db.inTransaction(async client => {
+      let dbQuery
+
+      dbQuery = {
+        text: `DELETE FROM active_local_delivery_units WHERE probation_area_code = $1`,
+        values: [probationAreaCode],
+      }
+
+      await client.query(dbQuery)
```

Comment:
would merge the query and the call here.

Reply:
I thought with it being a 2 step operation that i would need to call
 **await client.query(dbQuery)**
first to delete and then again to perform the insert??

Reply:
done